front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx

```diff
@@ -203,13 +199,40 @@ const ImportTrainScheduleConfig = ({
         const formattedArrivalTime = `${startDate} ${arrivalTime}`;
         const formattedDepartureTime = `${startDate} ${departureTime}`;
 
+        let stopFor: number | undefined;
+
+        const arrivalDate = new Date(`${startDate}T${arrivalTime}`);
+        const departureDate = new Date(`${startDate}T${departureTime}`);
+        if (ocpType === 'stop') {
+          if (arrivalTime && departureTime) {
+            if (arrivalTime === departureTime) {
+              stopFor = 0;
+            } else {
+              stopFor = Math.round((departureDate.getTime() - arrivalDate.getTime()) / 1000);
+            }
+          } else if (departureTime) {
+            stopFor = 0;
+          }
+        } else if (ocpType === 'pass') {
+          if (arrivalTime && departureTime && arrivalTime !== departureTime) {
+            // Incorrect data, it shouldn't happen on a ocpType pass, skip step
+            return null;
+          }
+          if (isLastOcp && departureTime) {
```

Comment:
Why do you need to check departureTime here ? :)

Reply:
it is indeed useless :D thanks